src/main/stackpack/resources/waitingfordata.md

```diff
@@ -0,0 +1,36 @@
+## The Splunk Topology StackPack is waiting for your action
+
+Provisioning of StackState configuration is done. We are now ready to process your data. If you haven't installed the StackState API Integration Agent please do so. After the installation of the API Integration Agent  we can configure the Splunk Topology integration.
```

Comment:
Are there two spaces between "Agent  we"?

Reply:
removed the extra space.